successstories/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/html">
+<head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-167873271-1"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+
+        function gtag() {
+            dataLayer.push(arguments);
+        }
+
+        gtag('js', new Date());
+        gtag('config', 'UA-167873271-1');
+    </script>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="ScholarX is an exclusive 6-month program aimed at providing
+            mentoring support to a selected pool of high Potential undergraduate students
+            based in Sri Lanka ideally by a Sri Lankan expat currently engaged with one of
+            the world’s top universities or Fortune 500 companies. It’s our free premium
+            mentoring platform by Sri Lankans for Sri Lankans working towards creating a culture
+            of knowledge and expertise sharing without the limitation of geographical borders.">
+    <title>ScholarX Success Stories | Sustainable Education Foundation</title>
+    <!-- Favicon -->
+    <link href="/assets/img/brand/favicon.png" rel="icon" type="image/png">
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700" rel="stylesheet">
+    <!-- Icons -->
+    <link href="/assets/vendor/nucleo/css/nucleo.css" rel="stylesheet">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" rel="stylesheet">
+    <!-- Argon CSS -->
+    <link type="text/css" href="/assets/css/argon.css?v=1.0.1" rel="stylesheet">
+    <!-- global CSS -->
+    <link type="text/css" href="/assets/css/global-custom.css" rel="stylesheet">
+
+    <link type="text/css" href="./styles.css" rel="stylesheet">
+
+    <link href="https://fonts.googleapis.com/css?family=Montserrat&display=swap" rel="stylesheet">
+</head>
+
+<body style="font-family: 'Montserrat', sans-serif;">
+<header class="header-global">
+    <!--nav bar goes here-->
+    <div id="navbar"></div>
+</header>
+
+
+<section class="story-section ">
+    <div class="container">
+        <h1 class="display-3 text-center main-topic">ScholarX Success Stories</h1>
+        <br>
+        <div id="cards">
+        </div>
+    </div>
+</section>
+<footer class="footer">
+    <div id="footer"></div>
+    <!--footer goes here -->
+</footer>
+<!-- to use the dynamic footer, use this instead of above tag => *<div id="footer"></div>* -->
+</body>
+
+<script id="template" type="x-tmpl-mustache">
+    {{#arr}}
```

Comment:
let's change this to stories instead of arr

Reply:
sure